forum/settings.py

```diff
@@ -108,12 +109,15 @@
         'NAME': os.environ.get('POSTGRES_DB'),
         'USER': os.environ.get('POSTGRES_USER'),
         'PASSWORD': os.environ.get('POSTGRES_PASSWORD'),
-        'HOST': os.environ.get('POSTGRES_HOST'),
+        'HOST': os.environ.get('POSTGRES_DOCKER_DB'),
```

Comment:
Maybe it makes sence to leave 'HOST': os.environ.get('POSTGRES_HOST') as it is for local using and add 'POSTGRES_DOCKER_DB' option in the comment with warning about neccessity of changing enviroment key whtn building docker volumes?

Reply:
If I leave POSTGRES_HOST, I have an issue with database connection. It can be a local error. Someone should checkout my branch and test it on POSTGRES_HOST.